src/amoc_users_sup.erl

```diff
@@ -28,7 +28,9 @@ init([]) ->
 
     SupFlags = {RestartStrategy, MaxRestarts, MaxSecondsBetweenRestarts},
 
-    Restart = temporary,
+    Restart = temporary, %% A temporary child process is never restarted
+    %% sending exit(Child,shutdown) first. if process doesn't stop withing
+    %% the shutdown timeout, than killing it brutally with exit(Child,kill).
```

Comment:
```suggestion
    %% the shutdown timeout, than kill it by exit(Child,kill).
```
```suggestion
    %% the shutdown timeout, it will be killed it by exit(Child,kill).
```